DCHdigi/src/DCHdigi_v01.cpp

```diff
@@ -101,6 +99,10 @@ DCHdigi_v01::operator()(const edm4hep::SimTrackerHitCollection& input_sim_hits,
                     const edm4hep::EventHeaderCollection&   headers) const {
   // initialize seed for random engine
   this->PrepareRandomEngine(headers);
+  // Gaussian random number generator used for the smearing of the z position, in cm!
+  std::normal_distribution<double> m_gauss_z_cm{0., m_z_resolution.value() * MM_TO_CM};
+  // Gaussian random number generator used for the smearing of the xy position, in cm!
+  std::normal_distribution<double> m_gauss_xy_cm{0., m_xy_resolution.value() * MM_TO_CM};
```

Comment:
```suggestion
  std::normal_distribution<double> gauss_z_cm{0., m_z_resolution.value() * MM_TO_CM};
  // Gaussian random number generator used for the smearing of the xy position, in cm!
  std::normal_distribution<double> gauss_xy_cm{0., m_xy_resolution.value() * MM_TO_CM};
```

You might want to rename them since "m_"  prefix you probably reserve for the member variables